app/helpers/import_helper.rb

```diff
@@ -13,14 +13,14 @@ def save_or_log(record)
     return if record.save
 
     error_message = "Could not save #{record.class.name} with id #{record.id}"
-    if Rollbar.configuration.enabled
-      Rollbar.error(error_message, errors: record.errors.full_messages)
+    if ENV['SENTRY_DSN'].present?
+      Sentry.capture_message(error_message, level: 'warning', extra: { errors: record.errors.full_messages })
     else
       Rails.logger.error "#{error_message}: #{record.errors.full_messages.join(',')}"
     end
   rescue StandardError => e
-    if Rollbar.configuration.enabled
-      Rollbar.error(e, record_class: record.class.name, record_id: record.id)
```

Comment:
Why not use `SENTRY_ENVIRONMENT`? Or can't we just send the excpetion and it silently ignores it when unconfigured?